salmon/model.py

```diff
@@ -381,13 +365,13 @@ def r_squared(self, X = None, y = None, adjusted = False, **kwargs):
         sse = ((y - pred.iloc[:,0]) ** 2).sum()
         ssto = ((y - y.mean()) ** 2).sum()
 
-        if adjusted:
+        if not adjusted:
             numerator = sse
             denominator = ssto
         else:
-            numerator = sse / (len(y) - len(self.X_train_.columns) - 2)
-            denominator = ssto / (len(y) - 1)
-            
+            numerator = sse / (len(y) - len(self.X_train_.columns) - 1)
```

Comment:
Just checked this code. The way the model is setup, this actually needs to be:
`numerator = sse / (len(y) - len(model.coef_) - 1)`

This is because we do special things for the intercept and so it isn't always counted properly via `len(self.X_train_.columns)`.

Reply:
Can you make this change in your pull request please? And good initial find by the way!